test/e2e/singlecluster/visibility_test.go

```diff
@@ -30,6 +30,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	"sigs.k8s.io/kueue/apis/visibility/v1alpha1"
```

Comment:
```suggestion
```

This is duplicated with `visibility "sigs.k8s.io/kueue/apis/visibility/v1alpha1"`.